### PR TITLE
[ZH] Update `Game mode/osu!taiko`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Game_mode/osu!taiko/zh.md [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-outdated_since: 832959f10a5d38fc57ecb291d6cb2f2a3af89d73
-outdated_translation: true
----
-
 # osu!taiko
 
 ![osu!taiko 游玩](/wiki/shared/taiko-gameplay.jpg "osu!taiko 的界面")
 
-osu!taiko 是 osu! 中的游戏模式，从日本节奏游戏[太鼓达人](https://zh.wikipedia.org/wiki/Taiko_no_Tatsujin) （在北美的发行名为[Taiko: Drum Master](https://zh.wikipedia.org/wiki/Taiko:_Drum_Master)）中借鉴了*一些*元素。
+osu!taiko 是 osu! 中的，基于日本节奏游戏[太鼓达人](https://zh.wikipedia.org/wiki/Taiko_no_Tatsujin)系列（在北美的发行名为[Taiko: Drum Master](https://zh.wikipedia.org/wiki/Taiko:_Drum_Master)）的游戏模式。
```
Comment (RushFTK) [inline wiki/Game_mode/osu!taiko/zh.md:5]:
```suggestion
osu!taiko 是基于日本节奏游戏[太鼓达人](https://zh.wikipedia.org/wiki/Taiko_no_Tatsujin)系列（在北美的发行名为[Taiko: Drum Master](https://zh.wikipedia.org/wiki/Taiko:_Drum_Master)）的 osu! 游戏模式。
```

File: wiki/Game_mode/osu!taiko/zh.md [hunk 2]
```diff
@@ -25,24 +20,25 @@ osu!taiko 是 osu! 中的游戏模式，从日本节奏游戏[太鼓达人](http
...
-  - 如果有故事板元素，则会占用除重要游戏元素外的所有屏幕空间
+  - 没有实现打击气球或谱面分歧这样的游戏机制（只有基本元素）
+- 背景显示差异
+  - 谱面的视频或背景图像只会占用屏幕下半部分
+  - 如果谱面含有故事板，故事板则会占据屏幕下半部分
+
+对于 osu!taiko 模式的新手来说：
 
-对于 osu!taiko 模式的新手来说，屏幕分成两个不同部分；上半部分含有游戏元素，而下半部分含有谱面的背景图片或视频。上半部分有血条，一开始是空的，必须填到至少 50% 满才能通过难度。血条下方的中上区域左侧有一个太鼓，旁边是传送带，将打击物件从右侧送到左侧。期间会通过太鼓旁的一个双层白色圆圈，作为判定圈使用。太鼓上方是太鼓的动画吉祥物，会随获得的判定结果有不同表现，也会改变上半部分的背景颜色。最后，右上角有常见的分数、准确度和谱面计时器。
+屏幕会分成两个不同部分；上半部分含有游戏元素，而下半部分含有谱面的背景图片或视频。上半部分有血条，它不像其他游戏模式一样，而在开始时是空的，必须填到至少 50% 满才能通过难度。血条下方的中上区域左侧有一个太鼓，旁边是传送带，将打击物件从右侧输送到左侧。期间，打击物件会通过太鼓旁的判定圈。太鼓上方是太鼓的动画吉祥物（也被叫做 pippidon 或咚），它会根据谱面是否处于 Kiai 段落，以及连击数达到里程碑时做出相应的反应。如果在 Kiai 时间内出现失误，则传送带背景会变换颜色。最后，右上角有常见的分数、准确度和谱面计时器。
```
Comment (RushFTK) [inline wiki/Game_mode/osu!taiko/zh.md:33]:
```suggestion
屏幕会分成两个不同部分；上半部分含有游戏元素，而下半部分含有谱面的背景图片或视频。在上半部分，有一个与其他游戏模式不一样的血条，其在开始时是空的，必须填到至少 50% 满才能通过难度。血条下方的中上区域左侧有一个太鼓，旁边是传送带，将打击物件从右侧输送到左侧。期间，打击物件会通过太鼓旁的判定圈。太鼓上方是太鼓的动画吉祥物（也被叫做 pippidon 或咚），它会根据谱面是否处于 Kiai 段落，以及连击数达到里程碑时做出相应的反应。如果在 Kiai 时间内出现失误，则传送带背景会变换颜色。最后，右上角有常见的分数、准确度和谱面计时器。
```

File: wiki/Game_mode/osu!taiko/zh.md [hunk 3]
```diff
@@ -25,24 +20,25 @@ osu!taiko 是 osu! 中的游戏模式，从日本节奏游戏[太鼓达人](http
...
 
-请注意，osu!taiko 中是禁用掉血的，因此只有打击物件会影响血条。当游玩区域为空时，也不会断连。与 osu! 不同，每 50 个连续打击设定一个连击里程碑。在获得以 10 为倍数的连击后，会增加基本得分，但在 100 连击时会停止加分（加分的最大连击限制）。如果断连了，则会重置到基本得分。在 *Kiai 时间*期间，打击成功会给玩家当前得分多 20% 的分数。打击获得的分数可以在右上角准确度下方的红色数字看到。
+请注意，osu!taiko 中，是不存在自然掉血的，因此，只有打击物件会影响到血条。不像 osu! 模式一样，osu!taiko 模式的连击里程碑是每 50 连击出现一次。打击物件的分数会从连击数大于 10 之后开始增长，但这个增长趋势会在连击数大于 100 后停止（被当前最大连击限制）。如果断连，打击物件的分数会回到基础分。在 *Kiai 时间*内，玩家能获得额外的 20% 物件分数。通过打击物件获得的单一物件分能够在右上角准确度下方的红色数字看到。
```
Comment (RushFTK) [inline wiki/Game_mode/osu!taiko/zh.md:35]:
```suggestion
请注意，osu!taiko 中不存在自然掉血，因此，只有打击物件会影响到血条。不像 osu! 模式一样，osu!taiko 模式的连击里程碑是每 50 连击出现一次。打击物件的分数会从连击数大于 10 之后开始增长，但这个增长趋势会在连击数大于 100 后停止（能获得的最高奖励分的最大连击）。如果断连，打击物件的分数会回到基础分。在 *Kiai 时间*内，玩家能获得额外的 20% 物件分数。通过打击物件获得的单一物件分能够在右上角准确度下方的红色数字看到。
```

Comment (RushFTK) [reply wiki/Game_mode/osu!taiko/zh.md:35]: >...The base score will be boosted after obtaining a combo that is a multiple of 10, ***but stops at 100 (max combo limit for the boost)*** If the combo is broken, the boost resets back to the base score. ...
guess "max combo" here refers to the 100.
not means if a diff's max combo lower 500 the bonus won't reach 100.
我认为这里更多指100是最高连击奖励分。

File: wiki/Game_mode/osu!taiko/zh.md [hunk 4]
```diff
@@ -102,20 +98,20 @@ osu!taiko 的默认控制方式如下：
...
 ## osu!taiko 作图
 
-- 红色音符指的是常规打击圈，
-  - 红色大音符需要重音音效。
-- 蓝色音符需要打击圈上带有哨声/掌声音效，
-  - 蓝色大音符需要重音和哨声音效。
-- 滑条代表长黄条（也被称作滚动条）。
-- 转盘代表拨浪鼓。
+- 红色音符对应普通的圆圈，
+  - 红色大音符对应使用了镲 (Finish) 音效的圆圈。
```
Comment (RushFTK) [inline wiki/Game_mode/osu!taiko/zh.md:102]:
```suggestion
  - 红色大音符需添加镲 (Finish) 音效。
```

File: wiki/Game_mode/osu!taiko/zh.md [hunk 5]
```diff
@@ -102,20 +98,20 @@ osu!taiko 的默认控制方式如下：
...
+- 蓝色音符对应使用了哨声/掌声 (Whistle/Clap) 音效的圆圈，
+  - 蓝色大音符对应同时使用了镲 (Finish) 和哨声/掌声音效的圆圈。
```
Comment (RushFTK) [inline wiki/Game_mode/osu!taiko/zh.md:104]:
```suggestion
  - 蓝色大音需同时添加镲 (Finish) 和哨声/掌声音效。
```

**Review by RushFTK — CHANGES_REQUESTED** [review]
Review translations

File: wiki/Game_mode/osu!taiko/zh.md (hunk 5 above)
Comment (YumeMuzi) [reply wiki/Game_mode/osu!taiko/zh.md:104]: 你自己都掉东西了？符呢？

File: wiki/Game_mode/osu!taiko/zh.md (hunk 4 above)
Comment (YumeMuzi) [reply wiki/Game_mode/osu!taiko/zh.md:102]: 我个人认为这里翻译成对应关系更好，因为音效和音符颜色并不是原文中的那种关系。

Comment (RushFTK) [reply wiki/Game_mode/osu!taiko/zh.md:102]: 那换个语序呢，例如:  
普通圆圈为红色音符  
添加音效为红色大音符  
(手机不好回复review)

Comment (YumeMuzi) [reply wiki/Game_mode/osu!taiko/zh.md:102]: 其实差不多的
no difference

**Review by RushFTK — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**